pkgs/applications/misc/zscroll/default.nix

```
@@ -2,16 +2,16 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "zscroll";
-  version = "1.0";
+  version = "2.0.1";
 
   # don't prefix with python version
   namePrefix = "";
 
   src = fetchFromGitHub {
     owner = "noctuid";
     repo = "zscroll";
-    rev = "v${version}";
-    sha256 = "0rf9m1czy19hzpcp8dq9c5zawk0nhwfzzjxlhk9r2n06lhb81ig5";
+    rev = "${version}";
```

```suggestion
    rev = version;
```
Unnecessary string conversions should be avoided.
https://nixos.org/manual/nixpkgs/stable/#sec-syntax

---

Fixed both issues